draft-irtf-cfrg-aegis-aead.md

```diff
@@ -1846,19 +1848,19 @@ Alternatively, the associated data can be fed into a collision-resistant KDF, su
 
 AEGIS nonces match the size of the key. AEGIS-128L and AEGIS-128X feature 128-bit nonces, offering an extra 32 bits compared to the commonly used AEADs in IETF protocols. The AEGIS-256 and AEGIS-256X variants provide even larger nonces. With 192 random bits, 64 bits remain available to optionally encode additional information.
```

Comment:
`current IETF protocols` or `at the time of writing` as this may change.

Reply:
Maybe not "current", as IETF protocols using primitives with a larger nonce may already exist.

"at the time or writing" has been added, then.